packages/components/date-picker/src/use-date-range-picker.ts

```diff
@@ -290,6 +290,7 @@ export function useDateRangePicker<T extends DateValue>({
       style: {
         ...props.style,
         maxWidth: "fit-content",
+        flexShrink: 0,
```

Comment:
do we have to set it here instead of `packages/core/theme/src/components/date-input.ts`?

Reply:
I have set it here as there was no slot just for `start-input`.

Reply:
Regarding the scrolling, could we wrap the `start-input` and `end-input` inside a `div`? So that we could achieve something like this

![image](https://github.com/nextui-org/nextui/assets/8769408/90587f6c-d905-4526-8ad9-40bc5df31831)

Reply:
> I have set it here as there was no slot just for start-input.

just a few lines below

```tsx
className: dateInputSlots.input({
  class: clsx(classNames?.input, props?.className),
}),
```

> Regarding the scrolling, could we wrap the start-input and end-input inside a div? So that we could achieve something like this

I think it's fine. We should only scroll the input. The scrollbar can look better tho.

Reply:
> 
> just a few lines below
> 
> ```tsx
> className: dateInputSlots.input({
>   class: clsx(classNames?.input, props?.className),
> }),
> ```

I guess we might not need `flexShrink: 0` as we are adding a scrollbar.


> 
> I think it's fine. We should only scroll the input.

If we dont add a `div` wrapper then the complete `input-wrapper` will be scrolled and by default the calendar button is not visible, we need to scroll to press the calendar button.

![image](https://github.com/nextui-org/nextui/assets/8769408/a31f01bd-bcdc-4ba7-b30d-b838d5021604)

![image](https://github.com/nextui-org/nextui/assets/8769408/81236bf3-7b10-4d6d-8345-fe497521ea38)

> 
> The scrollbar can look better tho.

I have used the default scrollbar like the components `Table` or  `ScrollShadow`.

Is their a custom scrollbar already present or do you want to me implement one?

Apart from the default scrollbar, the only scrollbar i found being used in the project is the below one? Do you want to me make use of this?
![image](https://github.com/nextui-org/nextui/assets/8769408/b2fb50dc-7aec-472e-b4df-4baba35a1950)

Reply:
use the default scrollbar first. we may refactor later.

Reply:
I guess we might need to increase the height of the component as just applying scrollbar hides the input as shown below:-
![image](https://github.com/nextui-org/nextui/assets/8769408/1175844c-b833-4e50-a681-9185ac6ef3a3)

Do you want me to increase the height? If yes could you tell me the values to which i need to increase?
Currently the `innerWrapper` height is set `h-6` and the size of the `inputWrapper` is `h-8` `h-10` `h-12` for size `sm` `md` & `lg` respectively.

Reply:
i think not. the height shouldn't be changed.

Reply:
If the height is not changed then the text will not be visible.

Reply:
the scrollbar shouldn't look like this. may double check with other pages. I rmb it was quite thin.